agent/command/s3_get.go

```diff
@@ -365,8 +370,9 @@ func (c *s3get) fetchAndExtractTarball(ctx context.Context, f *os.File) error {
 
 func (c *s3get) createPailBucket(ctx context.Context, comm client.Communicator, httpClient *http.Client) error {
 	opts := pail.S3Options{
-		Region: c.Region,
-		Name:   c.Bucket,
+		Region:               c.Region,
+		Name:                 c.Bucket,
+		VerifyChecksumSha256: c.RequireChecksumSha256,
```

Comment:
This got renamed in the Pail update but is functionally exactly the same. I'll update it here after the pail PR gets merged in.